webpack.config.js

```diff
@@ -9,7 +9,6 @@ const path = require('path');
 
 const BUILD_DIR = path.resolve(__dirname, 'src/client/public');
 const APP_DIR = path.resolve(__dirname, 'src/client/app');
-const COMMON_DIR = path.resolve(__dirname, 'src/common');
```

Comment:
Why? Don't we need stuff in src/common?

Reply:
We don't use COMMON_DIR anywhere in our webpack config (we don't bundle the stuff in that directory).

Reply:
We bundle in the `TimeInterval` class.

Reply:
We don't explicity point the webpack config at the src/common directory. We use it on the front-end, so webpack auto-bundles it.

Reply:
Makes sense!